doc/tech/README.md

```diff
@@ -90,23 +93,70 @@ Any specification work created within CNTT **must** conform to set of principles
 Within the framework of the Common Telecom cloud infrastructure vision, there are four levels of documents needed to describe the components, realize the practical application of the systems and qualify the resulting cloud infrastructure. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Conformance**.
 
 <p align="center"><img src="./figures/tech_scope.png" alt="scope" title="Document Types" width="100%"/></p>
-<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+<p align="center"><b>Figure 1:</b> Documentation Scope of CNTT</p>
 
 <a name="3.1"></a>
-## 3.1 Specification Types
+## 3.1 Functional Scope
+In terms of the functional scope of the CNTT documentation, in order to target the project goals as described above, we are focussed on:
+- Functional interfaces between infrastructure and workloads
+- Functional interfaces between workload management and infrastructure management
+- Functional capabilities of the cloud infrastructure
```

Comment:
I would prefer stating the scope as:
- Functional capabilities of the cloud infrastructure and the infrastructure management (as we are defining requirements for both infrastructure, e.g., software and hardware profiles, capabilities that infrastructure should provide, etc. and infrastructure management, security, reliability, functional requirements, etc.)
- Functional interfaces between infrastructure and infrastructure management
- Functional interfaces between workloads and workload management
(I'm not fully understand what're the interfaces between workload management and infrastructure management... And I guess we are not focusing on interfaces between workload and infrastructure now, e.g., memory copy, SR-IOV, accelerator utilization interface, etc.)

Reply:
Thanks - updated in https://github.com/cntt-n/CNTT/pull/1330/commits/32933f5d6f6d5fc1c6f8d278f9afd2f46f17c0e9